site/content/integrate/apps/interactivity/_index.md

```diff
@@ -97,14 +97,15 @@ The type of fields is the same as for Modal forms.
 
 All fields include:
 
-| Name          | Type      | Description                                                             |
-| :------------ | :-------- | :---------------------------------------------------------------------- |
-| `name`        | string    | Key to use in the values field of the call, and as part of the command. |
-| `type`        | FieldType | The type of the field.                                                  |
-| `is_required` | bool      | Whether the field needs to be filled.                                   |
-| `description` | string    | Text to show on the description line on autocomplete.                   |
-| `hint`        | string    | Text to show on the hint line on autocomplete.                          |
-| `label`       | string    | Label to name the field in autocomplete.                                |
+| Name          | Type      | Description                                                                                                                                                    |
+| :------------ | :-------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `name`        | string    | Key to use in the values field of the call, and as part of the command.                                                                                        |
+| `type`        | FieldType | The type of the field.                                                                                                                                         |
+| `is_required` | bool      | Whether the field needs to be filled.                                                                                                                          |
+| `description` | string    | Text to show on the description line on autocomplete.                                                                                                          |
+| `hint`        | string    | Text to show on the hint line on autocomplete.                                                                                                                 |
+| `label`       | string    | Label to name the field in autocomplete.                                                                                                                       |
+| `position`    | int       | (Optional) Positional argument (can be provided without a --flag). `If >0`, indicates the position this field is in. `If =-1`, it is considered the last argument. |
```

Comment:
I don't know whether I got the formatting correct on this, @larkox?

Reply:
It seems to render just fine. May be a bit unaligned on the source, but it shows fine on the tables.